### ISSUE TITLE
a11y CRITICAL: `auro-radio` - color contrast on disabled state (both alaska theme and hawaiian theme)

### ISSUE THREAD

**Issue by sun-mota** [AlaskaAirlines/auro-formkit#596]
### What browser client(s) are you seeing the issue on?

- [ ] Firefox
- [ ] Chrome
- [ ] Safari
- [ ] Microsoft Edge

### What devices platform(s) are you seeing the issue on?

- [ ] Desktop
- [ ] Mobile

### Steps to reproduce

![Image](https://github.com/user-attachments/assets/27750dbb-9902-4ed7-922b-4fcab536cced)
current disabled value: var(--ds-basic-color-texticon-disabled, #d0d0d0)

  Rule: color-contrast
      Impact: serious
      Elements must meet minimum color contrast ratio thresholds (https://dequeuniversity.com/rules/axe/4.10/color-contrast?application=axeAPI)
      
      Issue target: span
      Context: `<span slot="legend">Form label goes here</span>`
      Fix any of the following:
        Element has insufficient color contrast of 1.54 (foreground color: #d0d0d0, background color: #ffffff, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1

### Reproducible on the Auro doc site

- [x] I am able to reproduce this issue on the [Auro doc site](https://auro.alaskaair.com).

**Comment by blackfalcon** [comment]
Adding @blackfalcon as a participant

See parent issue

- https://github.com/AlaskaAirlines/auro-formkit/issues/497

**Comment by jason-capsule42** [comment]
Need to retest this using the new Alaska/Hawaiian/Loyalty themes

**Comment by sun-mota** [comment]
> Need to retest this using the new Alaska/Hawaiian/Loyalty themes

Will create a separate ticket for hawaiian theme.

**Comment by sun-mota** [comment]
aria-disabled should be added to indicate that the element is disabled.

**Comment by sun-mota** [comment]
confirmed that it's resolved in the latest layoutSupport
